components/inputs/input-number.js

```diff
@@ -0,0 +1,83 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { getUniqueId } from '../../helpers/uniqueId.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { inputLabelStyles } from './input-label-styles.js';
+import { inputStyles } from './input-styles.js';
+import { offscreenStyles } from '../offscreen/offscreen.js';
+
+class InputNumber extends LitElement {
+
+	static get properties() {
+		return {
+			autocomplete: { type: String },
+			autofocus: { type: Boolean },
+			disabled: { type: Boolean },
+			label: { type: String },
+			labelHidden: { type: Boolean, attribute: 'label-hidden' },
+			max: { type: Number },
+			min: { type: Number },
+			name: { type: String },
+			placeholder: { type: String },
+			required: { type: Boolean },
+			step: { type: Number },
+			value: { type: Number },
+			width: { type: String }
```

Comment:
I don't think we need `width`, since they can set it using inline styles `<d2l-input-text style="width:400px">`.